src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -72,7 +72,7 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
     useState<TActivityTransitionState>("IDLE");
   const [isOpen, setIsOpen] = useState(false);
   const [expanded, setExpanded] = useState<string | false>(false);
-  const [userSection, setUserSection] = useState<IUserSection>("clinic");
+  const [userSection, setUserSection] = useState<IUserSection>("laboratory");
```

Comment:
Is this that it should remain clinic for now.

Reply:
Oh Sorry. I forgot to restore it. I set it to "clinic" now.